pkg/loadbalancer/proxy_test.go

```diff
@@ -76,9 +76,9 @@ func Test_generateConfig(t *testing.T) {
 			want: &proxyConfigData{
 				HealthCheckPort: 32000,
 				ServicePorts: map[string]data{
-					"IPv4_80": data{
-						Listener: endpoint{Address: "0.0.0.0", Port: 80},
-						Cluster:  []endpoint{{"10.0.0.1", 30000}, {"10.0.0.2", 30000}},
+					"IPv4_80_TCP": data{
+						Listener: endpoint{Address: "0.0.0.0", Port: 80, Protocol: string(v1.ProtocolTCP)},
```

Comment:
You could just say `Protocol: "TCP"`; the templates already depend on the constants having specific expected values

Reply:
lazy copy and paste